core-services/prow/02_config/shiftstack/prune/_prowconfig.yaml

```diff
@@ -0,0 +1,12 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
```

Comment:
If we were to add the `jira` plugin, also make sure to add `jira/invalid-bug` to  `missingLabels`.

Reply:
why?

Reply:
https://github.com/openshift/release/blob/b669424/core-services/prow/02_config/shiftstack/gazelle/_prowconfig.yaml#L11